definitions/divorce/json/CaseEvent.json

```diff
@@ -439,8 +439,8 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "AwaitingDecreeNisi",
     "PostConditionState": "AwaitingLegalAdvisorReferral",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/dn-submitted",
-    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/dn-submitted",
```

Comment:
why `aboutTo`?

Reply:
The dn-submitted callback should handle Decreenisi Answer generation and email sending to petitioner now, so to save case data we have moved this callback from CallBackURLSubmittedEvent to CallBackURLAboutToSubmitEvent

Reply:
@alexwakeman I remember I've seen a story you created about sending notification always after event, not in "about to" stage.

What was the reason?

Reply:
In general it is better to notify after CCD is updated rather than before in case the operation fails. But if the callback is also updating the case it needs to happen before the CCD operation.

Reply:
ok